detection-rules/open_redirect_newegg.yml

```diff
@@ -0,0 +1,18 @@
+name: "Open redirect: Newegg"
+description: |
+  Message contains use of the Newegg open redirect, but the sender is not Newegg. This has been exploited in the wild.
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and any(body.links, .href_url.domain.root_domain == 'newegg.com' and .href_url.path =~ '/rts/go2.aspx')
```

Comment:
looks like the `x` url param is what is used for the redirect website. might include that value?